db/migrate/20201024121156_create_court_date_instance_from_casa_case_court_date_column.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCourtDateInstanceFromCasaCaseCourtDateColumn < ActiveRecord::Migration[6.0]
+  def up
+    CasaCase.all.each do |cc|
+      if cc.court_date
+        CourtDate.create!(date: cc.court_date, casa_case_id: cc.id)
+      end
+    end
+  end
+
+  def down
+    CourtDate.all.each do |cd|
+      cd.destroy!
+    end
+  end
+end
```

Comment:
@matisnape we are using [After Party](https://github.com/theSteveMitchell/after_party) for one-time data population tasks. After party tasks run after the data migrations run.

Reply:
Yeah, I've seen it. I started with what I know and build up from there :)